Makefile

```diff
@@ -86,6 +86,14 @@ test-features-integration: check-warn ## Run the BDD feature tests in the integr
 		--define="use_shared_resources=${USE_SHARED_RESOURCES}" \
 		$(FEATURE_TEST_ARGS)
 
+integration-test-with-custom_tag:
+	@echo "Running feature tests in the integration environment ${TF_WORKSPACE_NAME}"
+	behave --define="integration_test=true" --tags=@custom_tag \
+		--define="env=$(TF_WORKSPACE_NAME)" \
+		--define="account_name=$(ENV)" \
+		--define="use_shared_resources=${USE_SHARED_RESOURCES}" \
+		$(FEATURE_TEST_ARGS)
```

Comment:
! this seems very handy, could you add a line to the readme specifying this option alongside where it mentions the full integration test run?

Reply:
Readme now updated 👍